sdlib/d/gc/page.d

```diff
@@ -796,22 +831,30 @@ private:
 					continue;
 				}
 
-				// If the cycle do not match, all the elements are dead.
-				if ((w & 0xff) != gcCycle) {
+				auto slotsWithMeta = e.slabMetadataFlags.rawContent[0];
+
+				// If the cycle do not match, all the elements are dead. If no
+				// metadata exists, then we can safely clear the whole thing.
+				if (slotsWithMeta == 0 && (w & 0xff) != gcCycle) {
 					deadExtents.insert(e);
 					continue;
 				}
 
 				auto oldOccupancy = e.slabData.rawContent[0];
 				auto newOccupancy = oldOccupancy & (w >> 8);
+				auto toRemove = oldOccupancy ^ newOccupancy;
```

Comment:
toRemove isn't the proper name here.

Reply:
Opposite of occupied... `evicted`? Is there something else you think is better?

Reply:
I went with evicted.